apps/definition/mix.exs

```diff
@@ -26,7 +26,11 @@ defmodule Definition.MixProject do
       {:ok, in_umbrella: true},
       {:norm, "0.10.4"},
       {:jason, "~> 1.1"},
+      {:uuid, "~>1.1"},
```

Comment:
would be interested to hear @bbalser 's take on this vs. `puid` as a more performant alternative.

Reply:
I benchmarked these and on initial go it did not seem faster.

iex(3)> Benchmark.measure(&UUID.uuid4/0)
0.039
iex(4)> Benchmark.measure(&UUID.uuid4/0)
0.031
iex(5)> Benchmark.measure(&UUID.uuid4/0)
0.027

vs

iex(3)> Benchmark.measure(&AlphanumPuid.generate/0)
0.044
iex(4)> Benchmark.measure(&AlphanumPuid.generate/0)
0.029
iex(5)> Benchmark.measure(&AlphanumPuid.generate/0)
0.038

Reply:
There might be ways to make a PUID spec that has the appropriate entropy, collision resistance, that is faster, but i would expect the gains to be marginal so I say we punt for now.